pkgs/development/libraries/agg/default.nix

```diff
@@ -8,8 +8,18 @@ stdenv.mkDerivation rec {
     url = "https://www.antigrain.com/${pname}-${version}.tar.gz";
     sha256 = "07wii4i824vy9qsvjsgqxppgqmfdxq0xa87i5yk53fijriadq7mb";
   };
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ autoconf automake libtool freetype SDL libX11 ];
+  nativeBuildInputs = [
+    pkg-config
+    autoconf
+    automake
+    libtool
```

Comment:
This could all be replaced with autoreconfHook.

Reply:
Seems to fail with
```
autoreconfPhase
autoreconf: export WARNINGS=
autoreconf: warning: autoconf input should be named 'configure.ac', not 'configure.in'
autoreconf: Entering directory '.'
autoreconf: configure.in: not using Gettext
autoreconf: running: aclocal --force 
aclocal: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:11: error: automatic de-ANSI-fication support has been removed
./lib/autoconf/general.m4:2437: AC_FATAL is expanded from...
/nix/store/ppilclv3vx3vp6zgazcrdgbidd3xz47y-automake-1.16.3/share/aclocal-1.16/obsolete.m4:26: AM_C_PROTOTYPES is expanded from...
configure.in:11: the top level
autom4te: error: /nix/store/34fsscs8hfwjjgnswgzar1s9y1ybmzn2-gnum4-1.4.18/bin/m4 failed with exit status: 1
aclocal: error: /nix/store/mjpjsvxd36zi4vl9gm8mj9bnir8lzc28-autoconf-2.71/bin/autom4te failed with exit status: 1